<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSolution</class>
 <widget class="QDialog" name="NewSolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1192</width>
        <height>695</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_welcome" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_welcome">
            <property name="text">
             <string>Welcome to Designer</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="label_gettingStarted">
                <property name="text">
                 <string>Getting Started</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_newSln">
                <property name="text">
                 <string>New solution</string>
                </property>
                <property name="icon">
                 <iconset resource="../Image.qrc">
                  <normaloff>:/Image/solutionNew.png</normaloff>:/Image/solutionNew.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_openSln">
                <property name="text">
                 <string>Open solution</string>
                </property>
                <property name="icon">
                 <iconset resource="../Image.qrc">
                  <normaloff>:/Image/solutionOpen.png</normaloff>:/Image/solutionOpen.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="label_recently">
                <property name="text">
                 <string>Recently used content</string>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLineEdit" name="lineEdit_search">
                  <property name="placeholderText">
                   <string>Search</string>
                  </property>
                  <property name="cursorMoveStyle">
                   <enum>Qt::LogicalMoveStyle</enum>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_earlierSolution">
                <property name="text">
                 <string>Earlier solution</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidget_earlerSln">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::CustomContextMenu</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
                <property name="viewMode">
                 <enum>QListView::ListMode</enum>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_cfgNewSln" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>39</number>
          </property>
          <property name="topMargin">
           <number>39</number>
          </property>
          <property name="rightMargin">
           <number>39</number>
          </property>
          <property name="bottomMargin">
           <number>39</number>
          </property>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>12</number>
            </property>
            <item row="4" column="0">
             <widget class="QLineEdit" name="lineEdit_slnPath"/>
            </item>
            <item row="7" column="0">
             <widget class="QLineEdit" name="lineEdit_slnName">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Put the solution in the same directory as the project</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit_proName"/>
            </item>
            <item row="9" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_lastStep">
                <property name="text">
                 <string>Last step</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_create">
                <property name="text">
                 <string>Create</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_slnPath">
              <property name="text">
               <string>Solution path:</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_slnName">
              <property name="text">
               <string>Solution Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_proName">
              <property name="text">
               <string>Project name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_cfgNewSln">
              <property name="text">
               <string>Configure the new solution</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1" alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="btn_path">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="checkBox_setPath">
              <property name="text">
               <string>Set it to the default directory</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Image.qrc"/>
 </resources>
 <connections/>
</ui>
